PerformanceTests/Sources/PerformanceTestRunner/Core/Model/Dimension.swift

```diff
@@ -0,0 +1,11 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+struct Dimension: Codable {
+    let name: String
+    let value: String
```

Comment:
Should we maybe comment these types & properties with their description from the source doc?